src/main/java/Duke.java

```diff
@@ -11,21 +14,29 @@ public static void main(String[] args) {
         System.out.println("-------------------------------");
         System.out.println("Hello! I'm Duke\n" +
                 "What can I do for you?");
-
-
         System.out.println("-------------------------------");
 
         String echo = " ";
+        ArrayList<String> addedList = new ArrayList<>();
 
         Scanner input = new Scanner(System.in);
         while (!echo.equals("bye")) {
             echo = input.nextLine();
-            if(!echo.equals("bye")){
+            if(!echo.equals("bye") && !echo.equals("list")){
```

Comment:
Perhaps can create a Parser class so that you do not need to add another `!echo.equals("newFeature")` when there is a new feature.

Reply:
Perhaps  `if(!echo.equals("bye")` can be removed since ` !echo.equals("bye"` is included in the while loop.